packages/typespec-test/test/anomalyDetector/generated/typespec-ts/src/rest/outputModels.ts

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT license.
 
 /** The response of entire anomaly detection. */
-export interface UnivariateUnivariateEntireDetectionResultOutput {
+export interface UnivariateEntireDetectionResultOutput {
```

Comment:
It seems we have name collisions here(that's why we have prefix with namespace before), but i didn't notice any name suffix with _N, could you share the link with me?